src/diffusers/models/transformers/transformer_2d.py

```diff
@@ -305,14 +307,15 @@ def _init_patched_inputs(self, norm_type):
 
         # PixArt-Alpha blocks.
         self.adaln_single = None
-        self.use_additional_conditions = False
         if self.config.norm_type == "ada_norm_single":
-            self.use_additional_conditions = self.config.sample_size == 128
+            self.use_additional_conditions = self.config.sample_size == 128 and self.use_additional_conditions
```

Comment:
The reason why we're having to to introduce `use_additional_conditions` as a constructor argument is because `self.config.sample_size != 128` for PixArt-Sigma right?

Reply:
The reason is we don't use this `use_additional_conditions` in PixArt-Sigma anymore.

Reply:
Okay makes sense.